clip-recap-pipeline/src/services/content-items.ts

```
@@ -0,0 +1,472 @@
+            
+            for (const obj of monthObjects.objects) {
+              if (items.length >= limit) {
+                hasMore = true;
+                break;
+              }
+              
+              try {
+                // Use customMetadata to filter by processing_status without full GET
+                const processingStatus = obj.customMetadata?.['processing-status'] as ContentItem['processing_status'];
+                if (processing_status && processingStatus !== processingStatus) {
+                  continue;
```

Biome (2.1.2)

[error] Comparing to itself is potentially pointless.



(lint/suspicious/noSelfCompare)